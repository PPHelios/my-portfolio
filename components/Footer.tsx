import React from 'react'
import SocialBar from './myComponents/SocialBar'

function Footer() {
  return (
    <footer className='py-10 px-5 flex gap-5 justify-between items-center flex-col md:flex-row'>
      <p>Â©2024 Haytham Atef. All rights reserved.</p>
      <SocialBar className='flex gap-5' />
    </footer>
  )
}

export default Footer