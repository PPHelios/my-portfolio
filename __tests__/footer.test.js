'use strict';

import renderer from 'react-test-renderer';
import Footer from '../components/Footer';
it('renders correctly', () => {
  const tree = renderer
    .create(<Footer/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders as an anchor when no page is set', () => {
  const tree = renderer.create(<Footer/>).toJSON();
  expect(tree).toMatchSnapshot();
});

