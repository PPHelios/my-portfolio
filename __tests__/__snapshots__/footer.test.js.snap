// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as an anchor when no page is set 1`] = `
<footer
  className="py-10 px-5 flex gap-5 justify-between items-center flex-col md:flex-row"
>
  <p>
    ©2024 Haytham Atef. All rights reserved.
  </p>
  <div
    className="flex gap-5"
  >
    <div>
      <a
        href="https://www.linkedin.com/in/haytham-atef/"
        target="_blank"
      >
        <img
          alt="LinkedIn"
          className="h-5 md:h-7"
          src="/link.svg"
        />
      </a>
    </div>
    <div>
      <a
        href="https://github.com/pphelios"
        target="_blank"
      >
        <img
          alt="GitHub"
          className="h-5 md:h-7"
          src="/git.svg"
        />
      </a>
    </div>
  </div>
</footer>
`;

exports[`renders correctly 1`] = `
<footer
  className="py-10 px-5 flex gap-5 justify-between items-center flex-col md:flex-row"
>
  <p>
    ©2024 Haytham Atef. All rights reserved.
  </p>
  <div
    className="flex gap-5"
  >
    <div>
      <a
        href="https://www.linkedin.com/in/haytham-atef/"
        target="_blank"
      >
        <img
          alt="LinkedIn"
          className="h-5 md:h-7"
          src="/link.svg"
        />
      </a>
    </div>
    <div>
      <a
        href="https://github.com/pphelios"
        target="_blank"
      >
        <img
          alt="GitHub"
          className="h-5 md:h-7"
          src="/git.svg"
        />
      </a>
    </div>
  </div>
</footer>
`;
